File: kitten-house-care/src/main/java/org/jesperancinha/housing/data/CareCenterDto.java
package org.jesperancinha.housing.data;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Getter;
import lombok.NoArgsConstructor;

@Builder
@Getter
@NoArgsConstructor
@AllArgsConstructor
public class CareCenterDto {
    private String name;

    private String address;

    private String refNumber;

    private String city;

    private String postCode;

    private String country;
}

File: kitten-house-care/src/main/kotlin/org/jesperancinha/housing/data/CareCenterDto.kt
package org.jesperancinha.housing.data

import lombok.AllArgsConstructor
import lombok.Builder
import lombok.Getter
import lombok.NoArgsConstructor

class CareCenterDto (
    private val name: String? = null,
    private val address: String? = null,
    private val refNumber: String? = null,
    private val city: String? = null,
    private val postCode: String? = null,
    private val country: String? = null
)